docs/quickstart.md

```diff
@@ -321,3 +383,43 @@ spark.sql("SELECT * FROM quickstart_table").show(false)
 
 org.apache.iceberg.exceptions.ForbiddenException: Forbidden: Principal 'quickstart_user' with activated PrincipalRoles '[]' and activated ids '[6, 7]' is not authorized for op LOAD_TABLE_WITH_READ_DELEGATION
 ```
+
+### Connecting with Trino
+
+To use a Polaris-managed catalog in [Trino](https://trino.io/), you can
+configure Trino to use the Iceberg REST API.
+
+Generate and export a `PRINCIPAL_TOKEN` per the [README](/README.md#creating-a-catalog-manually),
+but create a Polaris catalog with the following specifications:
+
+```
+curl -i -X POST -H "Authorization: Bearer $PRINCIPAL_TOKEN" -H 'Accept: application/json' -H 'Content-Type: application/json' \
+http://localhost:8181/api/management/v1/catalogs \
+-d '{"name": "polaris", "id": 100, "type": "INTERNAL", "readOnly": false, "storageConfigInfo": {"storageType": "FILE"}, "properties": {"default-base-location": "file:///tmp/path_to_catalog"}}'
+```
+
+You can then show catalogs with:
+
+```
+curl -i -X GET -H "Authorization: Bearer $PRINCIPAL_TOKEN" -H 'Accept: application/json' -H 'Content-Type: application/json' \
+http://localhost:8181/api/management/v1/catalogs
+```
```

Comment:
We don't have to go through the catalog creation process here, since it is covered above.